tests/flows/steps/test_create_new_aws_environment.py

```
@@ -14,11 +14,15 @@
     SkipStepError,
     UnexpectedStopError,
 )
-from swo_aws_extension.parameters import (
+from swo_aws_extension.parameters import (  # noqa: WPS235
```

Ruff (0.14.11)

Unused `noqa` directive (unknown: `WPS235`)

Remove unused `noqa` directive

(RUF100)